pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -120,3 +128,22 @@ func (r *TokenREST) GroupVersionKind(containingGV schema.GroupVersion) schema.Gr
 type getter interface {
 	Get(ctx genericapirequest.Context, name string, options *metav1.GetOptions) (runtime.Object, error)
 }
+
+// newContext return a copy of ctx in which new RequestInfo is set.
+func newContext(ctx genericapirequest.Context, resource, name string, gvk schema.GroupVersionKind) (genericapirequest.Context, error) {
+	oldInfo, found := genericapirequest.RequestInfoFrom(ctx)
+	if !found {
+		return nil, fmt.Errorf("missing requestInfo")
```

Comment:
I think this will cause problems since it's not an API error. Can you just return context in this case?

Reply:
Done.